js/webui.js

```diff
@@ -2223,9 +2224,12 @@ var theWebUI = {
 		if (!w)
 			return;
 		const offcanvas = $("#offcanvas-sidepanel");
-		if (theWebUI.settings["webui.list_table_min_height"]) {
+		if (theWebUI.settings["webui.side_panel_min_width"]) {
```

Comment:
This is a typo fix that we can also backport to v5.1.

Reply:
Yes, agreed! Can you submit a PR to master for this?